src/app/executions/task-info/task-info.component.html

```diff
@@ -49,6 +49,19 @@
       <cf-info-item class="flex-1-1-auto mnw-0" [prop]="properties['state_info']"></cf-info-item>
     </div>
 
+    <div class="d-flex flex-row">
+
+      <!-- started -->
+      <cf-info-item class="flex-1-1-auto" [prop]="properties['started_at']"></cf-info-item>
+
+      <!-- finished -->
+      <cf-info-item class="flex-1-1-auto" [prop]="properties['finished_at']"></cf-info-item>
+
+      <!-- duration -->
+      <cf-info-item class="flex-1-1-auto" [prop]="properties['duration']"></cf-info-item>
+
+    </div>
```

Comment:
A quick solution (although not perfect) is to set a base width for the items (with a width of 33% each) so they'll have to wrap but still keep the spacing.

1) Move all 6 `cf-info-item` (of started, finished, duration, created_at, updated_at, duration_total) inside one div (i.e. delete lines 63-65)
2) Create a new style in `styles.scss`:
```scss
.flex-1-1-33 {
  flex: 1 1 33%;
}
```
3) Change the `cf-info-item` class to that new class (i.e. 
```html
<cf-info-item class="flex-1-1-33" [prop]="properties['started_at']"></cf-info-item>
```
)

Reply:
Maybe I understand something wrong, but I can't do it with placing all `cf-info-item` elements to the one div. I saved the old way and changed style. Everything looks nice.
![image](https://user-images.githubusercontent.com/17599857/56815069-1f631800-6849-11e9-8d99-f425add4ce26.png)